Jobs/Engine/Geocoding/Provider/Google.cs

```diff
@@ -43,9 +43,12 @@ bool IGeoCoder.IsApiKeyRequired
 
         GeocoderLocation IGeoCoder.Geocode(PropertyLocation address)
         {
-            string queryAdress = string.Format(((IGeoCoder)this).UrlPattern, HttpUtility.UrlEncode(address.ToString()));
-
-            WebRequest request = WebRequest.Create(queryAdress);
+            string queryAddress = string.Format(((IGeoCoder)this).UrlPattern, HttpUtility.UrlEncode(address.ToString()));
+            if (!string.IsNullOrWhiteSpace(address.ZipCode))
+            {
+                queryAddress = string.Format("{0}&components=postal_code:{1}|country:DE", queryAddress, address.ZipCode);
+            }
+            WebRequest request = WebRequest.Create(queryAddress);
```

Comment:
Bitte wenns geht sinnvolle Leerzeilen (Absätze) einfügen und nicht immer alles unmittelbar aneinanderreihen.  Das hier geht grad noch aber bitte in Zukunft drauf achten. Danke.